### ISSUE TITLE
Hardware ?

### ISSUE THREAD

**Issue by msvp** [xwiki-labs/cryptpad#779]
Hello from France !

I'm using cryptpad since a while for myself.

I would like to know what would be the hardware specifications of a server for 20 users simultaneously.

Who can tell me ?

Many Thanks !

**Comment by ansuz** [comment]
We provide a basic answer in our admin installation guide: https://docs.cryptpad.fr/en/admin_guide/installation.html#hardware

**Comment by msvp** [comment]
Thanks, I've read it before, but for how many users simultaneous is it build ?

**Comment by ansuz** [comment]
For 20 users I think any computer the average techie person has lying around will be sufficient. For example, a Raspberry PI 3 should be more than sufficient despite the fact that a lot of the server's cryptographic operations (mostly signature validation) are less performant on ARM processors. As far as I remember we were able to support 1000+  users with 2GB RAM and 1 CPU (before we introduced serverside child processes as workers for better parallelism).
